src/dsp_tools/commands/ingest_xmlupload/upload_files/upload_files.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+
+from loguru import logger
+from lxml import etree
+
+from dsp_tools.cli.args import ServerCredentials
+from dsp_tools.commands.ingest_xmlupload.bulk_ingest_client import BulkIngestClient
+from dsp_tools.commands.ingest_xmlupload.upload_files.filechecker import FileChecker
+from dsp_tools.commands.ingest_xmlupload.upload_files.upload_failures import UploadFailureDetails
+from dsp_tools.models.exceptions import InputError
+from dsp_tools.utils.connection import Connection
+from dsp_tools.utils.connection_live import ConnectionLive
+from dsp_tools.utils.xml_utils import remove_comments_from_element_tree
+
+
+def upload_files(
+    xml_file: Path,
+    creds: ServerCredentials,
```

Comment:
it would be theoretically possible to use `ServerCredentials` as a class field in `BulkIngestClient`, but I don't see the point of it. Do you think there would be an advantage?

Reply:
Consistency

Reply:
1. Are there any clients that have the ServerCredentials as a field? I'm not aware that our clients have that.
2. Even if so, the info contained in ServerCredentials is not needed in BulkIngestClient:

The info needed in BulkIngestClient:
    dsp_ingest_url: str
    token: str
    shortcode: str

The info in ServerCredentials:
    user: str
    password: str
    server: str
    dsp_ingest_url: str

So 75% of the info of ServerCredentials is not needed in BulkIngestClient!